app/src/main/java/corp/kairos/adamastor/UserContext.java

```diff
@@ -15,17 +21,35 @@
 public class UserContext {
     private String contextName; //The name of the context
     private List<AppDetail> contextApps; //The list of apps associated with this context
+    private Location location;
+    private GregorianCalendar init;
+    private GregorianCalendar end;
 
     public UserContext(String contextName, List<AppDetail> contextApps) {
         this.contextName = contextName;
         this.contextApps = contextApps;
     }
-
     public String getContextName() {
         return contextName;
     }
 
     public List<AppDetail> getContextApps() {
         return contextApps;
     }
+    private void removeApp(AppDetail app){
```

Comment:
Considering there now is a method to remove an app, any reason for not having one that can add?